cloudprovider/aliyun/aliyun_provider_test.go

```diff
@@ -0,0 +1,70 @@
+package aliyun
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/check.v1"
```

Comment:
Normally we do `. "gopkg.in/check.v1"`, so you don't need to refer `check` after.